spec/scenic/adapters/postgres_spec.rb

```diff
@@ -26,6 +26,15 @@ module Adapters
           expect(view.name).to eq("greetings")
           expect(view.materialized).to eq true
         end
+
+        it "raises an exception if the version of PostgreSQL is too old" do
+          adapter = Postgres.new
+          allow(ActiveRecord::Base.connection)
```

Comment:
The adapter now allows us to pass in a connection, so let's take advantage of that. Something like:

```ruby
connection = double("Connection", supports_materialized_views?: false)
adapter = Postgres.new(connection)
```

I'd prefer to stub the actual method we're calling here rather than test the integration between the connection and the decorator by passing a version. We can unit test the version checks in the connection decorator specs.